src/main/resources/templates/admin/email.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE HTML>
+<html xmlns:th="http://www.thymeleaf.org">
+<body>
+
+<span th:replace="~{/admin/header :: top}"/>
+
+<a href="/admin/">Home</a> -
```

Comment:
Remove the hyphen at the end since this is the only element in the list of parent pages.